Subsystems/cmdUtil/SendUdp.c

```diff
@@ -23,99 +23,107 @@
 #include "SendUdp.h"
 
 #ifdef WIN32
-    #pragma warning (disable:4786)
-    #include <winsock2.h>
-    #include <ws2tcpip.h>
-    #include <windows.h>
-    typedef int socklen_t;
+#pragma warning(disable : 4786)
```

Comment:
Has this been built on WIN32 to ensure that these pragma changes still work with the spacing changed?  Sometimes these can be picky.

Reply:
Nope.  I'm actually inclined to remove the special WIN32 logic.  We don't test it and nothing else does special handling to support it.

Reply:
Although I am curious what works and doesn't in WSL... but that's a different issue.